site/app/config/config.yml

```diff
@@ -160,9 +162,17 @@ fos_user:
     user_class: 'Librecores\ProjectRepoBundle\Entity\User'
     service:
         user_manager: librecores_user_manager
+        mailer: fos_user.mailer.twig_swift
     from_email:
         address:        'noreply@librecores.org'
         sender_name:    'LibreCores'
+    registration:
+            confirmation:
+                from_email: # Use this node only if you don't want the global email address for the confirmation email
+                    address:        'noreply@librecores.org'
+                    sender_name:    'LibreCores'
+                enabled:    true # change to true for required email confirmation
+                template:   'emailer.html.twig' #custom emailer template
```

Comment:
Instead of assigning a custom template name, I would prefer you to override the default FOSUserBundle templates as in https://github.com/librecores/librecores-web/pull/193

Also add a plain-text version of the email for better support of mobile and command-line clients.

Reply:
You could use: "LibreCores is a project by The Free and Open Source Silicon Foundation C.i.C., 71-75 Shelton Street, London, WC2H 9JQ, United Kingdom"